utility/utils.py

```diff
@@ -1191,7 +1191,7 @@ def install_start_kafka(rgw_node, cloud_type):
     """
     log.info("install kafka broker for bucket notification tests")
     if cloud_type == "ibmc":
-        wget_cmd = "curl -o /tmp/kafka.tgz http://10.245.4.4/kafka_2.13-2.8.0.tgz"
+        wget_cmd = "curl -o /tmp/kafka.tgz https://10.245.4.4/kafka_2.13-2.8.0.tgz"
```

Comment:
As we need to manually download the binaries and then store it in the file server, would it make sense to name the archive as `kafka.tgz` and then have a soft link to any given version.

It would avoid us changing the code everytime a new release needs to be tested.